__tests__/app.test.js

```diff
@@ -77,13 +77,21 @@ describe("my express project", () => {
     describe("PATCH: /api/articles/:article_id", () => {
         it("200 - updates article by id", () => {
             const id = 1;
-            
-            return request(app).
+            let beforePatch = 0;
+            let afterPatch = 0;
+            return db.query(`SELECT votes FROM articles WHERE article_id=$1`, [id]).then(({ rows }) => {
```

Comment:
this is very clever but not really necessary
You can see the data in the `test-data` files - you can just use this!

Reply:
Thanks Doug. Do you want me to change it? or can I MERGE now?

Reply:
I still have some comments incoming!

There are some things to do before merging please!